### PR TITLE
III-4970 events serice provider

### PR DESCRIPTION
**Pull request by steftrenson**
Best reviewed commit-by-commit as the classes are moved it becomes harder to see if the dependencies are still the same.

### Changed
- `EventCommandHandlerProvider` is PSR-11 compliant.
- `EventEditingServiceProvider` is PSR-11 compliant.
- Share `PlaceRepository::class` directly instead of the string `place_repository`.
- `EventGeoCoordinatesServiceProvider` is PSR-11 compliant.
- `EventHistoryServiceProvider` is PSR-11 compliant.
- `EventJSONLDServiceProvider` is PSR-11 compliant.
- `EventPermissionServiceProvider` is PSR-11 compliant.
- `EventReadServiceProvider` is PSR-11 compliant.
- `EventRequestHandlerServiceProvider` is PSR-11 compliant.
- `ProductionServiceProvider ` is PSR-11 compliant.

---
[Ticket: https://jira.uitdatabank.be/browse/...](https://jira.uitdatabank.be/browse/III-4970)

### CODE REVIEW

File: bootstrap.php [hunk 1]
```diff
@@ -2,8 +2,17 @@
 
 use Broadway\CommandHandling\CommandBus;
 use Broadway\EventHandling\EventBus;
+use CultuurNet\UDB3\AggregateType;
```
Comment (steftrenson) [inline bootstrap.php:5]: Damn, this might cause some issue. PhpStorm orders them alphabetically during a move apparently.

Comment (steftrenson) [reply bootstrap.php]: Should be fixed.

File: app/Event/EventCommandHandlerProvider.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        ];
+    }
+
+    public function register(): void
+    {
+        $container = $this->getContainer();
+
+        $container->addShared(
+            UpdateSubEventsHandler::class,
+            function () use ($container): UpdateSubEventsHandler {
```
Comment (LucWollants) [inline app/Event/EventCommandHandlerProvider.php:42]: I think if you use [arrow functions](https://stitcher.io/blog/short-closures-in-php) the `use` keyword is no longer needed.

**Review by LucWollants — APPROVED** [review]
I already approved because I only have one small remark. I did also run all acceptance tests.

**Review by bertramakers — APPROVED** [review]
Nice work!

**Pull request merged**